### PR TITLE
Use instance profile for AWS creds

### PR DESCRIPTION
**Pull request by srhaber**
@lumoslabs/devops

### CODE REVIEW

File: lib/broadside/configuration/aws_configuration.rb [hunk 1]
```diff
@@ -18,6 +18,12 @@ class AwsConfiguration
 
     def initialize
       @credentials = Aws::SharedCredentials.new.credentials
+
+      # Try instance profile credentials if shared credentials don't exist
+      if @credentials.nil?
```
Comment (apurvis) [inline lib/broadside/configuration/aws_configuration.rb]: simpler:
```ruby
@credentials = Aws::SharedCredentials.new.credentials || Aws::InstanceProfileCredentials.new.credentials
```

File: lib/broadside/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Broadside
-  VERSION = '3.0.10'.freeze
+  VERSION = '3.0.11'.freeze
```
Comment (apurvis) [inline lib/broadside/version.rb]: 3.1.0 please

**Comment by srhaber** [comment]
@apurvis Made the changes, check again?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# 3.1.0
+-- Simplified syntax for AWS credentials
+
+# 3.0.11
```
Comment (apurvis) [inline CHANGELOG.md]: there is no 3.0.11 now?

**Comment by srhaber** [comment]
Removed.

**Pull request merged**